proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/resources/HubResponseTranslatorResource.java

```diff
@@ -50,14 +57,40 @@ public Response hubResponse(HubResponseTranslatorRequest hubResponseTranslatorRe
         final X509Certificate encryptionCertificate = X_509_CERTIFICATE_FACTORY.createCertificate(hubResponseTranslatorRequest.getConnectorEncryptionCertificate());
 
         final org.opensaml.saml.saml2.core.Response eidasResponse = eidasResponseGenerator.generate(hubResponseContainer, encryptionCertificate);
+
+        String hashedEidasDetails = hashResponseDetails(
```

Comment:
Are we going to move this specific step using `hashResponseDetails` and the `combineAttributeValues` into a separate class in saml-libs so we can make sure that saml-engine will be doing exactly the same thing?

Reply:
Yes, the hashResponseDetails is in saml-libs now. Moving the combineAttributeValues to saml-libs is a good idea. The proxy-node uses its own object type, so I think this a good thing to do once we have started looking at the Hub.